src/styles/index.ts

```diff
@@ -363,6 +363,10 @@ const styles = (theme: ThemeColors) =>
             verticalAlign: 'top',
         },
 
+        textBold: {
```

Comment:
Why do we need to write a new style for this? Are you sure there isn't an existing style you can use?

Reply:
removed